scripts/config/jest/index.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```

Comment:
That only makes sense if the actual implementation is moved to `src` as well.  A script implementation in `scripts` with an import from `src` is just unnecessarily confusing.  The config isn't useful to anything in `src` (at the moment), so why would it be in there?